src/app/core/directives/newCourse/newCourse.directive.ts

```diff
@@ -0,0 +1,24 @@
+import { Directive, ElementRef, Input, OnInit } from '@angular/core';
+@Directive({ selector: '[newCourse]' })
+export class NewCourseDirective implements OnInit {
+    private el;
+
+    constructor(el: ElementRef) {
+        this.el = el;
+    }
+
+    public ngOnInit() {
+        const currDate = new Date();
+        const courseDate = (new Date(this.courseDate));
+        if (courseDate < currDate) {
+            const timeDiff = Math.abs(courseDate.getTime() - currDate.getTime());
+            const diffDays = Math.ceil(timeDiff / (1000 * 3600 * 24));
+            if (diffDays <= 14 ) this.el.nativeElement.style.borderColor = 'green';
```

Comment:
Not so important, but I think that classes would be more useful here. Instead of inline styles